AzureCalling/Controllers/CallViewController.swift

```diff
@@ -64,6 +64,14 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
         updateToggleVideoButtonState()
 
+        localParticipantView.switchCamera = { [weak self] in
+            guard let self = self else {
+                return
+            }
+
+            self.callingContext.switchCamera { _ in }
+        }
```

Comment:
How about the localParticipantView in the grid? Do we need to update the switchCamera closure on the onQueueParticipantViewUpdate() function?

Reply:
No need. We have only a single instance of localParticipantView, so once we assign a proper switchCamera closure to localParticipantView it'll be functional regardless of the grid configuration (I've tested it).
Remote participants will still have the empty non-functional closure for switchCamera that does nothing.